Frends.Excel.ConvertToCSV/Frends.Excel.ConvertToCSV/Excel.cs

```diff
@@ -53,18 +52,17 @@ private static string ConvertDataSetToCSV(DataSet result, Options options, Cance
         foreach (DataTable table in result.Tables)
         {
             // Read only wanted worksheets. If none is specified read all.
-            if (options.ReadOnlyWorkSheetWithName.Contains(table.TableName) || options.ReadOnlyWorkSheetWithName.Length == 0)
+            if (options.ReadOnlyWorkSheetWithName != null && (options.ReadOnlyWorkSheetWithName.Contains(table.TableName) || options.ReadOnlyWorkSheetWithName.Length == 0))
```

Comment:
Won't this actually require the ReadOnlyWorkSheet to always be not null? I think that even if we see a null in this field then we should assume that we should handle all sheets.